### PR TITLE
Remove group assignment copying

### PR DESCRIPTION
**Pull request by nwalters512**
This feature has never worked (see #6256), so we're removing it entirely for now. We _may_ want to implement this correctly in the future.

Should we close the corresponding issue (#6256)? Or leave that around as a reminder that we might want to implement this in the future?

### CODE REVIEW

**Review by mwest1066 — APPROVED** [review]
I'm in favor of keeping an issue for this because it seems like a feature that people will obviously want. It may be clearer to close the old issue and open a new one (referencing the old one), however, with a brief description of the history here.

File: apps/prairielearn/src/pages/instructorAssessmentGroups/instructorAssessmentGroups.ejs [hunk 1]
```diff
@@ -133,36 +133,6 @@
...
-                  <div class="d-flex justify-content-end">
-                    <div class="form-group mb-0">
-                      <input type="hidden" name="__action" value="copy_assessment_groups">
-                      <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
-                      <button type="button" class="btn btn-secondary" data-dismiss="modal">Cancel</button>
-                      <button type="submit" class="btn btn-primary">Copy</button>
-                    </div>
-                  </div>
-              </div>
-              </form>
```
Comment (jonatanschroeder) [inline apps/prairielearn/src/pages/instructorAssessmentGroups/instructorAssessmentGroups.ejs:162]: Just found another "bug": this form should be closed one /div before this point... 🙂

**Review by jonatanschroeder — APPROVED** [review]

File: apps/prairielearn/src/pages/instructorAssessmentGroups/instructorAssessmentGroups.ejs (hunk 1 above)
Comment (nwalters512) [reply apps/prairielearn/src/pages/instructorAssessmentGroups/instructorAssessmentGroups.ejs:162]: I love the groups code so much 🙃

**Comment by nwalters512** [comment]
I opened #8911 to track a potential reimplementation of this.

**Pull request merged**